### ISSUE TITLE
Overhead from generating machine code throws off CPUID HZ

### ISSUE THREAD

**Issue by workhorsy** [workhorsy/py-cpuinfo#128]
The way we are getting the HZ is off, because of the overhead from setting up the byte code for each call to get_ticks.

~~~python
start = self.get_ticks() # Rebuilds the byte code and function
time.sleep(1)
end = self.get_ticks() # Rebuilds the byte code and function
ticks = (end - start)
~~~

Change it to be more like

~~~python
# Generate get_ticks_x86 first
func = self.get_ticks_func() # Rebuilds the byte code and function

start = func()
time.sleep(1)
end = func()
ticks = (end - start)

func.free()
~~~

**Comment by workhorsy** [comment]
On a AMD Ryzen 7 2700X CPU, the overhead is around 64232 ticks.

**Comment by workhorsy** [comment]
Fixed in #149
